sdks/python/apache_beam/io/fileio.py

```diff
@@ -37,6 +37,8 @@
 
 DEFAULT_SHARD_NAME_TEMPLATE = '-SSSSS-of-NNNNN'
 
+__all__ = ['FileSink']
```

Comment:
Why do we have FileBasedSource but FileSink? Make an alias (if it's too late to rename) and export exactly one version.